srcpkgs/hydrus/template

```diff
@@ -24,6 +24,7 @@ do_install() {
 	chmod +x hydrus_server.py hydrus_client.py
 
 	vmkdir usr/lib/hydrus
+	rm -rf hydrus/static/build_files
```

Comment:
Do not use `rm -f` unless you have to. The `-f` flag makes `rm` ignore all errors[^1]. We want errors. If for example the directory would be removed in a next release, the `rm` command should fail. This will notify you that upstream's structure has changed and that the `rm` command needs to be modified or removed.

The advantages of this approach aren't only theoretical. `hydrus/static/build_files` doesn't exist. The `rm -rf` line therefore does nothing. It properly fails to build without `-f`.

```suggestion
	rm -r static/build_files
```

[^1]: "ignore nonexistent files and arguments, never prompt"

Reply:
Oh thanks. I didn't noticed.